AsyncDisplayKit/ASControlNode.m

```diff
@@ -254,6 +269,7 @@ - (void)addTarget:(id)target action:(SEL)action forControlEvents:(ASControlNodeE
       [targetActions addObject:NSStringFromSelector(action)];
     });
 
+  _targetAdded = YES;
```

Comment:
That was my first idea to create an accessory method and basically return _controlEventDispatchTable.count. You think there will be any performance hits due to checking of the count property? That was the main reason I decided against it.